hadoop.md

```diff
@@ -1,2 +1,8 @@
 # Hadoop
 The platform uses Hadoop HDFS as a file system for Spark. For web-ui access to it use [the hadoop-name node address defined in the inventory file](https://github.com/nlesc-sherlock/emma/blob/master/ansible.md#provision) which is listening on port **50070**.
+
+To have password less ssh connections between nodes for the hadoop user, it is necessary to generate a ssh-key with the name **id_rsa**. It should be stored under **files** in the **hadoop** role.
```

Comment:
Maybe my text is miss leading, but for what I know:
"For setting up Hadoop on a cluster of machines, the master should be able to do a password-less ssh to start the daemons on all the slaves."

Are you saying it is anymore like that?